.github/workflows/build.yml

```diff
@@ -21,20 +21,39 @@ jobs:
         working-directory: far
         run: python tools/source_validator.py
 
+#------------------------------------------------------------------------------
+  changelog-checks:
```

Comment:
Do we really need to run this and hlf-checks separately on different VMs?

Yes, running things in parallel is good in general, but in this case both checks will likely take only a few seconds, way less than the preparations.
Maybe both can be steps in one workflow?

Reply:
Not at all. The only real advantage of having two separate checks is accounting. One can immediately see in the PR "Checks" section which check failed, `hlf` or `changelog`. It was the way to ask whether we want it for the price of two VMs. I like them combined as well.